libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -815,6 +847,21 @@ void AC_PosControl::set_alt_target_with_slew(const float& pos)
     input_pos_vel_accel_z(posf, zero, 0);
 }
 
+/// update_pos_offset_z - updates the vertical offsets used by terrain following
+void AC_PosControl::update_pos_offset_z(const float pos_offset_z)
```

Comment:
let's remove the "const" before "float pos_offset_z" here too (and anywhere else we find it).  The const really doesn't add any value I think and it makes the argument list longer than it needs to be.